velox/functions/prestosql/aggregates/tests/MinMaxTest.cpp

```diff
@@ -236,4 +236,34 @@ TEST_F(MinMaxTest, initialValue) {
   testAggregations({row}, {}, {"max(c1)"}, "SELECT -1");
 }
 
+TEST_F(MinMaxTest, decimalMinMax) {
+  /// TODO: Add vectorfuzzer support for decimals to use existing
```

Comment:
Can we open an issue for this? Add a description of the current blocker as well.